README.md

```diff
@@ -1,7 +1,7 @@
 # TEAMMATES Developer Web Site
 
-![Users Count](https://img.shields.io/badge/users-80K%2B-brightgreen.svg)
-![Universities Count](https://img.shields.io/badge/universities-1000%2B-green.svg)
+![Users Count](https://img.shields.io/badge/users-140K%2B-brightgreen.svg)
```

Comment:
Please do not change this line (users count) as it is not related to this issue.